src/test/java/org/assertj/core/error/ShouldContainThrowable_create_Test.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.error;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.Arrays.array;
+import static org.assertj.core.util.Throwables.getStackTrace;
+import static org.mockito.internal.util.collections.Sets.newSet;
+
+import java.util.Set;
+
+import org.assertj.core.internal.TestDescription;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for <code>{@link ShouldContainCharSequence#shouldContain(Throwable, CharSequence[], Set)})}</code> and
+ *  <code>{@link ShouldContainCharSequence#shouldContain(Throwable, CharSequence)})}</code>.
+ *
+ * @author Benoit Dupont
+ */
+@DisplayName("ShouldContainThrowable create")
+class ShouldContainThrowable_create_Test {
+  @Test
+  void should_create_error_message_with_escaping_percent() {
+    // GIVEN
+    RuntimeException actual = new RuntimeException("You know nothing %");
+    // WHEN
+    String errorMessage = ShouldContainCharSequence.shouldContain(actual, "You know nothing % Jon Snow")
+                                                   .create(new TestDescription("TEST"));
+    // THEN
+    assertThat(errorMessage).isEqualTo("[TEST] %n" +
```

Comment:
Replace `assertThat` by `then` from `BDDAssertions`.

Reply:
I just discovered BDDAssertions and saw that it's encouraged in the contributing guidelines. Is it to _allow to skip test execution when assumptions are not met_ as described in https://assertj.github.io/doc/#assertj-core-3.14.0-bdd-assumptions ?

Reply:
yes but I'm not sure what the use case is here, can you clarify?

Reply:
What I meant is: should we use BDDAssertions when we use the _given_ method to be able to skip the test execution when assumptions are not met, or use BDDAssertions by default instead of _assertThat_ as a general coding guideline ?